### PR TITLE
implemented use of enumerate columns in postgresql

### PR DESCRIPTION
**Pull request by vitormattos**
Enumerate in PostgreSQL is a new type and a separate SQL is required to implement it for this reason in alter table and in alter column is necessary runs a new query to identify if the type already exists, if the custom type dont exists,  the new type is created.

### CODE REVIEW

**Comment by mvrhov** [comment]
@robmorgan: What's missing to get this one merged?
Except of rebasing and squashing commits to 2 docs and the implementation itself.

File: src/Phinx/Db/Adapter/PostgresAdapter.php [hunk 1]
```diff
@@ -484,6 +512,19 @@ public function dropColumn($tableName, $columnName)
...
+        $this->dropTypeByName($tableName . '_' . $columnName . '_enum');
+    }
+    
+    protected function dropTypeByName($typeName)
+    {
+    	$this->startCommandTimer();
+    	$this->writeCommand('dropTypeByName', array($typeName));
+    	$sql = sprintf(
+    			'DROP TYPE IF EXISTS %s',
+    			$typeName
```
Comment (shadowhand) [inline src/Phinx/Db/Adapter/PostgresAdapter.php]: We have some mixing of tabs and spaces going on here.

**Comment by shadowhand** [comment]
Thank you for contributing! The branch you made this PR against is no longer accepting patches. This probably happened because we didn't merge your changes fast enough. :sob:

Please create a new PR against the current default branch if you want to see these changes merged.

**Comment by mvrhov** [comment]
@shadowhand: How quickly can this be merged if I rebase this against master?

**Pull request closed without merging**